src/transform.ts

```diff
@@ -16,6 +16,12 @@ export interface Transform {
    * Calculate new field(s) using the provided expresssion(s). Calculation are applied before filter.
    */
   calculate?: Formula[];
+
```

Comment:
Please remove this and use `as` where appropriate.  

We intentionally do not add deprecated properties to the Typescript interface so that they won't be generated in the schema.